pkg/sql/sqlbase/system.go

```diff
@@ -123,6 +123,21 @@ CREATE TABLE system.jobs (
 	INDEX (status, created),
 	FAMILY (id, status, created, payload)
 );`
+
+	SessionsTableSchema = `
+CREATE TABLE system.sessions (
+	id             SERIAL     PRIMARY KEY,
+	"hashedSecret" BYTES      NOT NULL,
+	username       STRING     NOT NULL,
+	"createdAt"    TIMESTAMP  NOT NULL DEFAULT now(),
+	"expiresAt"    TIMESTAMP  NOT NULL,
+	"revokedAt"    TIMESTAMP,
+	"lastUsedAt"   TIMESTAMP  NOT NULL DEFAULT now(),
+	"auditInfo"    STRING,
+	INDEX("expiresAt"),
+	INDEX("createdAt"),
+	FAMILY(id, "hashedSecret", username, "createdAt", "expiresAt", "revokedAt", "lastUsedAt", "auditInfo")
```

Comment:
Let's just as @bdarnell and he can make the call, I also see the merits and disadvantages of both approaches.  I went for the consistency angle.